package.json

```
@@ -74,7 +74,7 @@
     "@sentry/integrations": "^6.2.2",
     "@sentry/node": "^6.16.1",
     "@sentry/tracing": "^6.16.1",
-    "@typechain/ethers-v5": "^7.1.2",
+    "@typechain/ethers-v5": "^9.0.0",
```

```suggestion
    "@typechain/ethers-v5": "7.2.0",
```
https://github.com/energywebfoundation/iam-client-lib/blob/45db11a8679d1d0af31a5d5a9e84363fcebf1a1d/package.json#L112

---

if I won't upgrade to latest versions using `npm install` , there is constant dependency hell regardless of what I try, eg: 

While resolving: @typechain/ethers-v5@7.2.0
#11 197.0 npm ERR! Found: ethers@5.4.7
#11 197.0 npm ERR! node_modules/ethers
#11 197.0 npm ERR!   ethers@"5.4.7" from the root project
#11 197.0 npm ERR!   ethers@"^5.0.1" from ethereum-waffle@3.4.0
#11 197.0 npm ERR!   node_modules/ethereum-waffle
#11 197.0 npm ERR!     dev ethereum-waffle@"^3.4.0" from the root project
#11 197.0 npm ERR!   23 more (passport-did-auth, @ew-did-registry/did-document, ...)
#11 197.0 npm ERR! 
#11 197.0 npm ERR! Could not resolve dependency:
#11 197.0 npm ERR! peer ethers@"^5.1.3" from @typechain/ethers-v5@7.2.0
#11 197.0 npm ERR! node_modules/@typechain/ethers-v5
#11 197.0 npm ERR!   @typechain/ethers-v5@"7.2.0" from the root project
#11 197.0 npm ERR! 
#11 197.0 npm ERR! Conflicting peer dependency: ethers@5.6.1
#11 197.0 npm ERR! node_modules/ethers
#11 197.0 npm ERR!   peer ethers@"^5.1.3" from @typechain/ethers-v5@7.2.0
#11 197.0 npm ERR!   node_modules/@typechain/ethers-v5
#11 197.0 npm ERR!     @typechain/ethers-v5@"7.2.0" from the root project


So I rather suggest to upgrade rest of libs instead of downgrading this one.

---

Check this PR https://github.com/energywebfoundation/ssi-hub/pull/370. Problem is with docker and npm dedupe